controllers/v1/reportsController.js

```diff
@@ -3070,5 +3070,38 @@ submissionReport = async function (req) {
     }
   });
 };
+fetch = async function (req) {
```

Comment:
@borkarsaish65 we don't need a validation function for this endpoint?

Reply:
added